free5gc-smf-org-example/packagevariantset.yaml

```diff
@@ -0,0 +1,59 @@
+# Copyright 2023 The kpt Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: config.porch.kpt.dev/v1alpha2
+kind: PackageVariantSet
+metadata:
+  name: example
+spec:
+  upstream:
+    repo: catalog
+    package: foo
+    revision: v2
+  targets:
+  - repositories:
+    - name: cluster-01
+      packageNames:
+      - foo-01
+      - foo-02
+      - foo-03
+    template:
+      downstream:
+        packageExpr: "packageName + '-' + repository.labels['env']"
```

Comment:
`packageName` is not correct, this should be either `target.package` or `packageDefault` (which have the same value in this case).

Reply:
@johnbelamaric Does it address your comment?